tests/components/media_player/test_sync_helper.py

```diff
@@ -0,0 +1,135 @@
+"""The tests for the Sync Media player helper functions."""
+
+import pytest
+
+import homeassistant.components.media_player as mp
+from homeassistant.const import (
+    STATE_IDLE,
+    STATE_OFF,
+    STATE_ON,
+    STATE_PAUSED,
+    STATE_PLAYING,
+)
+
+
+class SyncMediaPlayer(mp.MediaPlayerEntity):
+    """Sync media player test class."""
+
+    def __init__(self, hass):
+        """Initialize the test media player."""
+        self.hass = hass
+        self._volume = 0
+        self._state = STATE_OFF
+
+    @property
+    def state(self):
+        """State of the player."""
+        return self._state
+
+    @property
+    def volume_level(self):
+        """Volume level of the media player (0..1)."""
+        return self._volume
+
+    @property
+    def supported_features(self):
+        """Flag media player features that are supported."""
+        return (
+            mp.const.SUPPORT_VOLUME_SET
+            | mp.const.SUPPORT_VOLUME_STEP
+            | mp.const.SUPPORT_PLAY
+            | mp.const.SUPPORT_PAUSE
+            | mp.const.SUPPORT_TURN_OFF
+            | mp.const.SUPPORT_TURN_ON
+        )
+
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        self._volume = volume
+
+    def volume_up(self):
+        """Turn volume up for media player."""
+        if self.volume_level < 1:
+            self.set_volume_level(min(1, self.volume_level + 0.2))
+
+    def volume_down(self):
+        """Turn volume down for media player."""
+        if self.volume_level > 0:
+            self.set_volume_level(max(0, self.volume_level - 0.2))
```

Comment:
Same here. What is the reason for choosing 0.2 over 0.1?

Reply:
See above

Reply:
Thanks.